frontend/src/advising/ManageStudents/ExpandedStudentView.tsx

```diff
@@ -49,6 +71,63 @@ const ExpandedStudentContainer = styled.div`
   padding: 30px;
 `;
 
+const CommentsHeader = styled.div`
```

Comment:
Can we move the comment component to a different file? I think a lot of this will be shared with the student view too. We might also want to add the ability to have comments in the non expanded student view